apps/participants/tables.py

```diff
@@ -18,7 +18,7 @@ class ParticipantTable(tables.Table):
 
     class Meta:
         model = Participant
-        fields = ("identifier", "created_at")
+        fields = ("get_platform_display", "identifier", "created_at")
```

Comment:
This will show `get_platform_display` in the table header. I think you need something like

```python
platform = columns.Column(accessor="get_platform_display")
```